docs/design/ocs-to-ocs-api-server.md

```diff
@@ -0,0 +1,201 @@
+# OCS to OCSs : API Server
+
+## Introduction
+This document defines the API server used for communication between OCS provider and OCS consumer clusters.
+
+
+## Tool
```

Comment:
Move all this to the bottom as an `## Alternatives` section.

Reply:
moved this section at the end.